src/applications/mhv/medications/containers/PrescriptionDetails.jsx

```diff
@@ -262,16 +323,27 @@ const PrescriptionDetails = () => {
 
   const handleModalClose = () => {
     dispatch(clearAllergiesError());
-    setPdfGenerateStatus(PDF_GENERATE_STATUS.NotStarted);
+    setPdfTxtGenerateStatus({
+      ...pdfTxtGenerateStatus,
+      status: PDF_TXT_GENERATE_STATUS.NotStarted,
```

Comment:
is there any reason we wouldn't want to reset the object entirely?
like this:
```
{
    status: PDF_TXT_GENERATE_STATUS.NotStarted,
    format: undefined,
    message: undefined,
  }
```

Reply:
No reason, except for format and message do not really change much so it's just not to trigger any extra refreshes.